release/src/main/scripts/build_release_candidate.sh

```diff
@@ -0,0 +1,220 @@
+git clone ${GIT_REPO_URL}
+cd ${BEAM_ROOT_DIR}
+git checkout ${RELEASE_BRANCH}
+
+echo '-------------------Generating Python Artifacts-----------------'
+cd sdks/python
+python setup.py sdist --format=zip
+cd dist
+
+svn co https://dist.apache.org/repos/dist/dev/beam
+mkdir beam/${RELEASE}/${PYTHON_ARTIFACTS_DIR}
+cp apache-beam-${RELEASE}.zip beam/${RELEASE}/python_test/apache-beam-${RELEASE}.zip
+cd beam/${RELEASE}/${PYTHON_ARTIFACTS_DIR}
+
+echo "------Signing Source Release apache-beam-${RELEASE}.zip------"
+gpg --armor --detach-sig apache-beam-${RELEASE}.zip
+
+echo "------Creating Hash Value for apache-beam-${RELEASE}.zip------"
+sha512sum apache-beam-${RELEASE}.zip > apache-beam-${RELEASE}.zip.sha512
+
+cd ..
+svn add ${PYTHON_ARTIFACTS_DIR}
+svn status
+echo "Please confirm these changes are ready to commit: [y|N] "
+read confirmation
+if [[ $confirmation != "y" ]]; then
+  echo "Exit without staging python artifacts on dist.apache.org."
+  rm -rf ~/${PYTHON_ARTIFACTS_DIR}
+  exit
+fi
+svn commit
+rm -rf ~/${PYTHON_ARTIFACTS_DIR}
+
+echo "==============Creating PR for Updating Website==============="
+cd ~
+mkdir ${LOCAL_WEBSITE_UPDATE_DIR}
+cd ${LOCAL_WEBSITE_UPDATE_DIR}
+mkdir ${LOCAL_PYTHON_DOC}
+mkdir ${LOCAL_JAVA_DOC}
+mkdir ${LOCAL_WEBSITE_REPO}
+
+echo "------------------Building Python Doc------------------------"
+cd ${LOCAL_PYTHON_DOC}
+pip install tox
+git clone ${GIT_REPO_URL}
+cd ${BEAM_ROOT_DIR}
+git checkout ${RELEASE_BRANCH}
+cd sdks/python && tox -e docs
+GENERATED_PYDOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_PYTHON_DOC}/${BEAM_ROOT_DIR}/sdks/python/target/docs/_build
+
+echo "----------------------Building Java Doc----------------------"
+cd ~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_JAVA_DOC}
+wget https://github.com/apache/beam/archive/release-${RELEASE}.zip
+unzip release-${RELEASE}.zip
+GENERATE_JAVADOC=~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_JAVA_DOC}/beam-release-${RELEASE}/sdks/java/javadoc
+
+echo "------------------Updating Beam Website---------------------"
+cd ~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_WEBSITE_REPO}
+git clone ${GIT_BEAM_WEBSITE}
+cd ${WEBSITE_ROOT_DIR}
+git checkout -b updates_release_${RELEASE}
+
+echo "..........Updating release version in _config.yml..........."
+sed -i -e "s/release_latest: [0-9].[0-9].[0-9]/release_latest: ${RELEASE}/g" _config.yml
+
+echo "..........Copying generated javadoc into beam-site.........."
+cp -r ${GENERATE_JAVADOC} src/documentation/sdks/javadoc/${RELEASE}
+echo "Updating javadoc current.md with latest release version number"
+sed -i -e "s/[0-9].[0-9].[0-9]/${RELEASE}/g" src/documentation/sdks/javadoc/current.md
+
+echo "............Copying generated pydoc into beam-site.........."
+cp -r ${GENERATED_PYDOC} src/documentation/sdks/pydoc/${RELEASE}
+echo ".....Updating pydoc current.md with latest release version number......"
+sed -i -e "s/[0-9].[0-9].[0-9]/${RELEASE}/g" src/documentation/sdks/pydoc/current.md
+
```

Comment:
Changes to downloads page is missing.
- Adding the new link, and archiving the previous versions.

Reply:
Right, will add.

Reply:
Addressed in commit https://github.com/apache/beam/pull/6059/commits/1b26a73a59afbf6f989c3c56da8f56b398bfa5d7

This step needs to be done manually since it asks for adding new section and attach JIRA link.